res/controllers/Denon-Prime-4-scripts.js

```diff
@@ -682,6 +682,7 @@ Prime4.padMode = {
 };
 
 Prime4.PadSection = function(deck, offset) {
+    const theContainer = this;
     components.ComponentContainer.call(this);
```

Comment:
I think you'll need to swap these two lines to fix the `TypeError`. Not sure though. In general you should not access `this` before calling the parent constructor.

Reply:
That didn't fix it. Still getting the same `TypeError` message

Reply:
mhmm ok, please reorder them anyways.